utils/make_release_notes.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+
+# file: make_release_notes.sh
+# description: makes a release notes template for a release of the cyclus stack
+# project
+
+set -e
+
+usage() {
+
+   cat <<EOF
+
+Script to generate release notes for the current release version relative to a previous version.
+
+usage: $0 <previous>  <version>
+
+       previous : the version string of the previous version (e.g., P.P.P) which provides the
+                  basis for determining what is new in the current release
+        version : the version string (e.g., V.V.V) to with which this package will be labeled.
+
+NOTE: all version strings must already be existing tags in both the Cyclus, Cycamore, and Cymetric repositories.
+
+The following environment variables must be set for this script to function:
+
+     CYCLUS_DIR : Environment variable CYCLUS_DIR must be set to the cyclus repository directory.
+   CYCAMORE_DIR : Environment variable CYCAMORE_DIR must be set to the cycamore repository directory.
+   CYMETRIC_DIR : Environment variable CYMETRIC_DIR must be set to the cymetric repository directory.
+
+EOF
+
+}
+
+die() {
+    echo -n >&2 "ERROR: "
+    echo >&2 "$@"
+    usage
+    exit 1
+}
+
+# special behavior for macosx
+sed_i() {
+    if [[ "`uname`" == "Linux" ]]; then
+	sed -i  "$1" "$2";
+    else
+    	sed -i '' "$1" "$2";
+    fi
+}
+
+HERE=$PWD
+
+# check input
+CYCLUS=${CYCLUS_DIR?"Environment variable CYCLUS_DIR must be set to the cyclus repository directory."}
+CYCAMORE=${CYCAMORE_DIR?"Environment variable CYCAMORE_DIR must be set to the cycamore repository directory."}
+CYMETRIC=${CYMETRIC_DIR?"Environment variable CYMETRIC_DIR must be set to the cymetric repository directory."}
+[ "$#" -eq 2 ] || die "Must provide the from version and to version (e.g., W.W.W -> X.X.X) as an argument"
+
+PREV=$1
+VERSION=$2
+echo "Making release notes template for Cyclus stack version $VERSION from $PREV. 
+
+Note, these version names must refer to *existing tags* in all repositories.
+"
+
+# cyclus summary
+cd $CYCLUS
+NCOMMITS=`git rev-list $PREV...$VERSION --count | tail -n1`
+SUMMARY=`git diff --stat $PREV...$VERSION | tail -n1`
+CYCLUSTXT="$NCOMMITS commits resulting in $SUMMARY"
+CYCLUSCONTRIB=`git log --format='%aN' $PREV...$VERSION | sort -u`
+cd $HERE
+
+# cycamore summary
+cd $CYCAMORE
+NCOMMITS=`git rev-list $PREV...$VERSION --count | tail -n1`
+SUMMARY=`git diff --stat $PREV...$VERSION | tail -n1`
+CYCAMORETXT="$NCOMMITS commits resulting in $SUMMARY"
+CYCAMORECONTRIB=`git log --format="%aN" $PREV...$VERSION | sort -u`
+cd $HERE
+
+# cymetric summary
+cd $CYMETRIC
+[[ `git tag` =~ $PREV ]] && CYMETRIC_PREV=$PREV || CYMETRIC_PREV=`git log --pretty=format:%H|tail -1`
```

Comment:
Whitespace on the sides of the pipe, ie `--pretty=format:%H | tail -1`.  Also, do you need the tail since I am pretty sure that `git log -1` will do the same thing.